docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/media/html5/index.md

```diff
@@ -154,21 +154,35 @@ This function begins tracking media events for a given media element. It takes a
 
 ---
 
+#### Filter Out Repeated Events Configuration (Optional)
+
+| Parameter | Type | Description | Default Value |
+|-----------|------|-------------|---------------|
```

Comment:
The `config.filterOutRepeatedEvents` parameter also accepts a boolean which can be used to enable/disable all repeated events regardless of the type, maybe worth adding a mention of that?

```
| `config.filterOutRepeatedEvents` | `boolean` | Whether to filter out seek start, seek end and volume change events. | `false` |
```